src/site/xdoc/server/config-spring-jpa-postgres.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.    
+-->
+<document>
+
+ <properties>
+  <title>Apache James Server 3 - Spring JPA With Postgres Configuration</title>
+ </properties>
+
+<body>
+
+
+    <section name="Introduction">
+      <p>Consult <a href="http://james.apache.org/server/config-system.html">System Configuration</a> to get template, some examples and hints for the configuration with JPA.</p>
+    </section>
+
+    <section name="Current Supported Sequence Database">
+      <p>James supports DERBY as a default database, you do not need to do any additional configuration if you are using Derby</p>
+    </section>
+
+    <section name="Spring JPA With Postgres Configuration">
+      <p>If you wish to use POSTGRES as a database, please follow those steps to configure it</p>
+      <subsection name="Step 1: Create database on Postgres">
+
+        <p>You need to install Postgres or run Postgres with Docker</p>
+
+        <p>Example to install it on Debian:</p>
+        <source>
+          sudo apt-get update && sudo apt-get install postgresql-9.4 postgresql-client-9.4
+        </source>
+        <p>Example to pull and launch Postgres on docker</p>
+        <source>
+          docker run --detach --name postgres library/postgres
+        </source>
+        <p>Postgres does not create database on the fly so you need to create the database manually. Here is the command to perform this action: </p>
+
+        <source>psql --username postgres --tuples-only --command "SELECT 1 FROM pg_database WHERE datname = '<strong><italic>database_name</italic></strong>'" | grep -q 1 || psql --username postgres --command "CREATE DATABASE <strong><italic>database_name</italic></strong>"</source>
```

Comment:
Ok I guess your "guess -q 1" is to make your command return a fail code if first SQL command fail. If you do that I think it's because you do not want to execute the second sql command if the first one fail. In this case, you should do:
```psql --username postgres --tuples-only --command "SELECT 1 FROM pg_database WHERE datname = 'database_name'" | grep -q 1 && psql --username postgres --command "CREATE DATABASE database_name".```

But maybe I did not understood what you want to do